sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterHelperExtensions.cs

```diff
@@ -29,6 +29,11 @@ public static TracerProviderBuilder AddAzureMonitorTraceExporter(this TracerProv
             var options = new AzureMonitorExporterOptions();
             configure?.Invoke(options);
 
+            if (options.StorageDirectory == null)
```

Comment:
What if customer provides an invalid location. Should we validate and fallback to default directory?

Reply:
do we provide option to turn this feature off?

Reply:
Not yet - this will be on by default. May be we can provide them an option which they can turn on/off for each telemetry type. Needs some discussion.